src/Jwt/JwtV2Validator.php

```diff
@@ -35,7 +35,7 @@ public function validateClaims(Token $token): void
         $this->baseValidator->validateClaims($token);
 
         $udb3Token = new Udb3Token($token);
-        if ($udb3Token->isAccessToken()) {
+        if ($udb3Token->isAccessToken() || $udb3Token->isClientToken()) {
```

Comment:
This should just stay "isAccessToken()`, there's no need to differentiate between user access tokens and client access tokens here I think.

And if there is, we should rename `isAccessToken()` to `isUserAccessToken()`.